File: tools-starter-swagger/src/main/java/cn/gjing/tools/swagger/core/PathSelect.java
package cn.gjing.tools.swagger.core;

import cn.gjing.tools.swagger.SwaggerBean;
import com.google.common.base.Predicate;

/**
 * @author Gjing
 **/
@SuppressWarnings("all")
interface PathSelect {
    Predicate<String> getPredicate(SwaggerBean swaggerBean,String pathPattern);
}

File: tools-starter-swagger/src/main/java/cn/gjing/tools/swagger/core/PathSelect.scala
package cn.gjing.tools.swagger.core

import cn.gjing.tools.swagger.SwaggerBean
import com.google.common.base.Predicate

/**
 * @author Gjing
 **/
trait PathSelect {
  def getPredicate(swaggerBean: SwaggerBean, pathPattern: String): Predicate[String]
}
